.github/workflows/ci.yml

```diff
@@ -1,6 +1,7 @@
 name: CI
 
-on: [pull_request]
+on:
+  pull_request:
```

Comment:
nit: why change this formatting?

Reply:
oups, ill revert, i was testing with `workflow_dispatch` and forgot to revert back